backend/src/worker.ts

```diff
@@ -40,7 +41,8 @@ async function main() {
     intrigueIdent,
     webscraper,
     savedSearch,
-    testProxy
+    testProxy,
```

Comment:
Given that subjack performs GET requests, we should add a test within `testProxy` to confirm that subjack's GET requests are signed.